ansible_collections/arista/avd/molecule/digital_twin/converge.yml

```diff
@@ -22,3 +29,45 @@
       delegate_to: 127.0.0.1
       ansible.builtin.import_role:
         name: arista.avd.eos_designs
+
+- name: Converge
+  hosts: DIGITAL_TWIN_SINGLE_SWITCH_FABRIC
```

Comment:
If the purpose of this fabric is just to test the generated topology file, then we don't need a non-digital-twin run, and we don't need eos_cli_config_gen stuff for it at all.
Really it could just have been a pytest.

Reply:
resolving for now - we will tackle moving all of this to pytest in one go.